examples/vision/object_detection_using_vision_transformer.py

```diff
@@ -0,0 +1,605 @@
+"""
+Title: End-to-end Object detection with transformers
+Author: Karan V. Dave
+Date created: 2022/03/27
+Last modified: 2022/03/27
+Description: A simple keras implementation for object detection using transformers.
+"""
+
+
+"""
+## Introduction
+
+ This example is a an implementation of the [Vision Transformer (ViT)]
+ (https://arxiv.org/abs/2010.11929)
```

Comment:
Format mkdown links on a single line each. Make sure to read and follow instructions from the [contributor guide](https://github.com/keras-team/keras-io/blob/master/contributor_guide.md).